integration-tests/src/test/resources/features/filter.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Filter tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    And add a Processor to the Bridge "mybridge" with body:
```

Comment:
I would put this step as part of scenario as it contains the specific scenario logic.

Reply:
Okay! Will do the necessary changes! Thanks